pkgs/tools/misc/wlc/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, argcomplete
+, python-dateutil
+, requests
+, pyxdg
+, pre-commit
+, pytest
+, pytest-cov
+, responses
+, twine
+}:
```

pull from pkgs scope
```suggestion
{ lib
, python3
}:

with python3.pkgs;
```
in this case, I think it's okay to use `with`, as this is a pretty plain python expression after